packages/forms/resources/views/components/checkbox-list.blade.php

```diff
@@ -152,39 +152,54 @@ class="filament-forms-checkbox-list-component-option-label flex items-center spa
                             "
                         @endif
                     >
-                        <input
-                            @if ($isBulkToggleable())
-                                x-on:change="checkIfAllCheckboxesAreChecked()"
-                            @endif
-                            wire:loading.attr="disabled"
-                            type="checkbox"
-                            value="{{ $optionValue }}"
-                            dusk="filament.forms.{{ $getStatePath() }}"
-                            {{ $applyStateBindingModifiers('wire:model') }}="{{ $getStatePath() }}"
-                            {{
-                                $getExtraAttributeBag()
-                                    ->class([
-                                        'text-primary-600 transition duration-75 rounded shadow-sm focus:border-primary-500 focus:ring-2 focus:ring-primary-500 disabled:opacity-70',
-                                        'dark:bg-gray-700 dark:checked:bg-primary-500' => config('forms.dark_mode'),
-                                        'border-gray-300' => ! $errors->has($getStatePath()),
-                                        'dark:border-gray-600' => (! $errors->has($getStatePath())) && config('forms.dark_mode'),
-                                        'border-danger-300 ring-danger-500' => $errors->has($getStatePath()),
-                                        'dark:border-danger-400 dark:ring-danger-400' => $errors->has($getStatePath()) && config('forms.dark_mode'),
-                                    ])
-                                    ->merge([
-                                        'disabled' => $isDisabled(),
+                        <div class="flex h-5 items-center">
```

Comment:
What is this fixed height for? 🙂

Reply:
> What is this fixed height for? 🙂

Without `h-5`
<img width="360" alt="Screenshot 2023-06-28 at 7 16 11 PM" src="https://github.com/filamentphp/filament/assets/315603/c818c994-3c0a-4129-8528-ade23957185c">

With `h-5`
<img width="310" alt="Screenshot 2023-06-28 at 7 16 29 PM" src="https://github.com/filamentphp/filament/assets/315603/ebb79ca9-edc6-4136-a4a1-de88eb3e29db">

😈

Reply:
Hmm why is the label too heigh without `h-5` though?
In my non-Filament project that I showed earlier in this thread, I got it working without a fixed height, to allow for wrapping labels.

Reply:
> Hmm why is the label too heigh without `h-5` though? In my non-Filament project that I showed earlier in this thread, I got it working without a fixed height, to allow for wrapping labels.

Maybe you have input margin

Reply:
Nope.

Reply:
> Nope.
> 
> Edit: actually yes lol. `mt-0.5` I think that's the better approach.

Ok. thank you for your time.

Reply:
> Nope.
> 
> Edit: actually yes lol. `mt-0.5` I think that's the better approach.

`mt-1`
<img width="313" alt="Screenshot 2023-06-28 at 10 07 57 PM" src="https://github.com/filamentphp/filament/assets/315603/ef150e3c-ae91-40ee-a575-fda07b1ac581">

`mt-0.5`
<img width="299" alt="Screenshot 2023-06-28 at 10 08 13 PM" src="https://github.com/filamentphp/filament/assets/315603/f717cda7-4a35-452b-835d-60e583a9c97d">

I'm happy with `h-5` approach. 👻

Reply:
Hmm neither looks centered. Could you figure out why?

Reply:
No

Reply:
Alright, then I'll have to take a look later.